earth_enterprise/src/fusion/portableglobe/cutter/htdocs/admin/js/database_view.js

```diff
@@ -95,6 +95,7 @@ gees.admin.databaseView = {
       var display = {};
       display.size = gees.tools.getDisplaySize(db.size);
       display.wms =  db.serve_wms ? this.wmsHoverText(db.target_path) : '';
+      display.def_db = db.default_database ? '<em>Default</em>' : '';
```

Comment:
Also very picky, but it would be nice if the "Default" tag was a different color than the "WMS" tag.